scripts/build_hydro_profile.py

```diff
@@ -64,7 +64,29 @@
 
 import atlite
 import geopandas as gpd
-from vresutils import hydro as vhydro
+import pandas as pd
+
+import country_converter as coco
+cc = coco.CountryConverter()
```

Comment:
Change of dependency in `environment.yaml` from PyPSA-EUR-Sec dep to PyPSA-EUR dep.